libs/embed/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nikolaik/python-nodejs:python3.10-nodejs16-slim
+FROM nikolaik/python-nodejs:python3.10-nodejs16-alpine
```

Comment:
Discussed this with @Cliftonz previously, I defintly think that will be the option, but we have a limitation right now because of the dynamic API_URL environment variables in the `CMD` command, we cant skip having the dev deps at this point removed until we will refactor how we inject them.